massa-bootstrap/src/lib.rs

```diff
@@ -346,6 +381,8 @@ struct BootstrapServer {
 }
 
 impl BootstrapServer {
+    // We don't want to leave on a fail of sending error to a bootstrap client.
+    #[allow(unused_must_use)]
```

Comment:
remove this allow, use `let_ = XXX `when we want to explictly ignore an important value